Core/StartupArgManager.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace LocalAdmin.V2.Core
+{
+    public sealed class StartupArgManager
+    {
+        /// <summary>
+        /// Path to startup arguments file.
+        /// </summary>
+        private const string StartupArgsPath = "laargs.txt";
+
+        /// <summary>
+        /// Merges Command-line arguments and arguments in <paramref name="StartupArgsPath"/>
+        /// </summary>
+        /// <param name="CMDArgs">Runtime Command-Line Arguments</param>
+        /// <returns>Merged Arguments</returns>
+        public static string[] MergeStartupArgs(string[] CMDArgs)
+        {
+            List<string> StartupArgs = new List<string>();
+            foreach (string arg in CMDArgs)
+            {
+                if (!StartupArgs.Contains(arg))
+                    StartupArgs.Add(arg);
+            }
+
+            foreach (string farg in File.ReadAllLines(Path.GetFullPath(StartupArgsPath)))
```

Comment:
This will throw an exception if the file is not found and prevent entire LA from starting.

Instead of bundling the file together with LA it should be created on startup if it doesn't exist.

Reply:
Fixed but it's a bit janky.